packages/realm-app-importer/src/RealmAppImporter.ts

```diff
@@ -0,0 +1,246 @@
+            method: "POST",
+            headers: { "content-type": "application/json" },
+            body
+        });
+        // Store the access and refresh tokens
+        const responseBody = await response.json();
+        this.accessToken = responseBody.access_token;
+
+        // Write the stitch config file
+        this.saveStitchConfig(
+            this.username,
+            responseBody.refresh_token,
+            responseBody.access_token
+        );
+    }
+
+    private saveStitchConfig(
+        username: string,
+        refreshToken: string,
+        accessToken: string
+    ) {
+        const stitchConfig = [
+            `public_api_key: ${username}`,
+            `refresh_token: ${refreshToken}`,
+            `access_token: ${accessToken}`
+        ];
+        fs.writeFileSync(
+            this.stitchConfigPath,
+            stitchConfig.join("\n"),
+            "utf8"
+        );
+    }
+
+    private async getProfile() {
+        if (!this.accessToken) {
+            throw new Error("Login before calling this method");
+        }
+        const url = `${this.baseUrl}/api/admin/v3.0/auth/profile`;
+        const response = await fetch(url, {
+            headers: { Authorization: `Bearer ${this.accessToken}` }
+        });
+        return response.json();
+    }
+
+    private async getGroupId() {
+        const profile = await this.getProfile();
+        if (typeof profile === "object" && profile.roles.length === 1) {
+            return profile.roles[0].group_id;
+        } else {
+            throw new Error("Expected user to have a role in a single group");
+        }
+    }
+
+    private async createApp(groupId: string, name: string) {
+        if (!this.accessToken) {
+            throw new Error("Login before calling this method");
+        }
+        const url = `${this.baseUrl}/api/admin/v3.0/groups/${groupId}/apps`;
+        const body = JSON.stringify({ name });
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                Authorization: `Bearer ${this.accessToken}`,
+                "content-type": "application/json"
+            },
+            body
+        });
+        return response.json();
+    }
+
+    private async createSecret(
+        groupId: string,
+        internalAppId: string,
+        name: string,
+        value: string
+    ) {
+        console.log(`Creating "${name}" secret`);
+        if (!this.accessToken) {
+            throw new Error("Login before calling this method");
+        }
+        const url = `${this.baseUrl}/api/admin/v3.0/groups/${groupId}/apps/${internalAppId}/secrets`;
+        const body = JSON.stringify({ name, value });
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                Authorization: `Bearer ${this.accessToken}`,
+                "content-type": "application/json"
+            },
+            body
+        });
+        if (response.ok) {
```

Comment:
`return response.ok;` maybe

Reply:
I changed this to throw if the response is not OK.